community/channels.rst

```diff
@@ -9,10 +9,10 @@ Note that some of these channels are run and moderated by members of the Godot c
 
 A brief overview over these and other channels is also available on the `Godot website <https://godotengine.org/community>`_.
 
-Q&A
+Forum
 ---
```

Comment:
The Build fails, because the underline of the title needs to match the length.

for example:
```
Forum
-----

Same problem for the second title change.
```